examples/nginx-role/roles/nginx-simple/defaults/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
 # defaults file for nginx
 nginx_test_message: This is a test message
-nginx_keepalive_timeout: 115
+nginx_webserver_port: 80
```

Comment:
Github code review doesn't allow you to leave comments on any line that was not touched (missed) in a change. This example fails if you change this default port value during your run. The smoke test task needs `url: http://localhost:{{ nginx_webserver_port }}/` in the `task/main.yml` file there.